tools/explorer/lib/model.py

```diff
@@ -128,35 +131,39 @@ class Match(object):
 
   The match is between EntityTypeFields objects and an EntityType object lists.
 
-  Attributes: field_list entity_type match_type
+  Attributes:
```

Comment:
this is args instead of attributed since GetIncrement is a def is not a class

Reply:
These attributes belong to the Match class.

Reply:
but is it not a class?